Sources/NIOHTTP2/ConnectionStateMachine/LocallyQuiescingState.swift

```diff
@@ -16,6 +16,6 @@
 /// the remote peer may not initiate new streams.
 ///
 /// This protocol should only be conformed to by states for the HTTP/2 connection state machine.
-protocol LocallyQuiescingState {
+protocol LocallyQuiescingState: ConnectionStateWithRole {
```

Comment:
I don't think the refinement logically applies here: it belongs on `ReceivingHeadersState`.